navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
Using a route resolver, would only make sense, IMO, if we were injecting vars into the template:

```js
 .when('/demo/:tenant/landing/', {
      vars: {
        title: 'Landing Page',
        navbar: {display: 'tenant'}
      },
      template: '<demo-landing-page something="foo"></demo-landing-page>',
      resolve: {
        foo: function($route) {
          // do stuff
          return Promise.resolve('some value for foo');
        }
      }
    })
```